x-pack/test/scalability/apis/api.telemetry.cluster_stats.no-cache.1600-dataviews.json

```diff
@@ -0,0 +1,47 @@
+{
+  "journeyName": "POST /api/telemetry/v2/clusters/_stats - no cache - 1600 dataviews",
```

Comment:
This use case struggles even for such a low number of users. If maps improves their collector, this should improve.